stockpile/roles/httpd/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+---
+  # Allow for capturing of the httpd configuration
+  # TODO: Remove hardcoding of indice values
+  - name: Get keepalive settings
+    shell:
+       grep "KeepAlive" /etc/conf/httpd.conf | cut -d ' ' -f 2
+    register: keep_alive
```

Comment:
Too generic will cause issues, I'd suggest to change it to perhaps httpd_conf_keep_alive